python/foglamp/common/json_parse.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Command line JSON parsing utility Class"""
```

Comment:
This is really the wrong place for this script. The foglamp/common directory is for things that are common to multiple FogLAMP components and are included in multiple places. This is really a utility script and as such should live somewhere like scripts/common I would say. @izoratti-dianomic can you please confirm?

Reply:
.../scripts/common is ok by me.

Reply:
If located in "scripts/common" it's not copied via "make install"

Currently Makefile does it for .sh files only:
```
install_common_scripts : $(SCRIPT_COMMON_INSTALL_DIR) $(COMMON_SCRIPTS_SRC)
        $(CP) $(COMMON_SCRIPTS_SRC)/*.sh $(SCRIPT_COMMON_INSTALL_DIR)
```